store/rootmulti/store.go

```
@@ -227,12 +227,12 @@ func (rs *Store) CacheMultiStoreWithVersion(version int64) (types.CacheMultiStor
 
 // Implements interface MultiStore
 func (rs *Store) GetStore(key types.StoreKey) types.Store {
-	panic("uncached KVStore is not supposed to be accessed directly")
+	return rs.stores[key]
```

should we panic with the missing key?

---

yes

---

seems we still need handle empty memory capability in [cacheMultiStore](https://github.com/crypto-org-chain/cronos/blob/8555bd947026b45011bd552952608a47b4194db2/versiondb/multistore.go#L39)

---

mem store is contained in the `transientStores`

---

mem store is contained in the `transientStores`

---

still have issue, because `MountMemoryStores` don't create the mem store yet, only `LoadLatest` does that.